api/src/test/java/org/openmrs/api/ProviderServiceTest.java

```diff
@@ -317,7 +317,7 @@ public void saveProviderAttributeType_shouldSaveTheProviderAttributeType() throw
 	 */
 	@Test
 	public void unretireProvider_shouldUnretireAProvider() throws Exception {
-		Provider provider = service.getProvider(1);
+		Provider provider = service.getProvider(2);
```

Comment:
Provider id 1 does not have a provider/person name because of which its giving a validation exception. 
log:
[Provider: providerId:1 providerName: ]
[Provider: providerId:2 providerName:[Mr. Horatio Test Hornblower, Mr. Horatio Peeter Hornblower2, Mr. John Peeter Hornblower3, Mr. adolf hitler] ]

org.openmrs.api.ValidationException: '[Provider: providerId:1 providerName: ]' failed to validate with reason: name: Provider Name or Person required, person: Provider Name or Person required